From d74ea1c8f4e983329bdae96b04e6958d207043c4 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Sun, 1 Jun 2014 15:52:40 +0200
Subject: [PATCH] tests: fix build when qreal!=double

Just use qreal when a qreal& argument is expected.

(cherry picked from commit 54f86b39d7e8d14d0ce915b9e98087f1d4d20b74)
---
 tests/GeoUriParserTest.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/GeoUriParserTest.cpp b/tests/GeoUriParserTest.cpp
index 20268e9..c0d91d3 100644
--- a/tests/GeoUriParserTest.cpp
+++ b/tests/GeoUriParserTest.cpp
@@ -71,7 +71,7 @@ void GeoUriParserTest::testGeoUri()
 
     GeoDataCoordinates coords = parser.coordinates();
 
-    double cLat = 0.0, cLon = 0.0, cAlt = 0.0;
+    qreal cLat = 0.0, cLon = 0.0, cAlt = 0.0;
     coords.geoCoordinates(cLon, cLat, cAlt, GeoDataCoordinates::Degree);
 
     QCOMPARE(ret, valid);
-- 
2.0.0.rc2

